classes/requests/helpers/class-kp-order-lines.php

```diff
@@ -342,6 +342,30 @@ private function process_coupons() {
 				}
 			}
 		}
+
+		/**
+		 * PW WooCommerce Gift Cards.
+		 */
+		if ( ! empty( WC()->session->get( 'pw-gift-card-data' ) ) ) {
+			$pw_gift_cards = WC()->session->get( 'pw-gift-card-data' );
+			foreach ( $pw_gift_cards['gift_cards'] as $code => $value ) {
+				$coupon_amount = intval( $value * -100 );
+				$gift_card_sku = apply_filters( 'kp_pw_gift_card_sku', esc_html__( 'gift_card', 'klarna-payments-for-woocommerce' ), $code );
```

Comment:
Since this will be used for the reference of the order line in Klarna, we probably need to change the name of this filter. Can we change it to "klarna_pw_gift..." instead? that will be easier for us to add in to both KCO and KOM as well in that case without having to think about it being prefixed with KP.